### ISSUE TITLE
Appending character to new scene "applies" desaturation filter

### ISSUE THREAD

**Issue by Eufalesio** [FlailingFog/KK-Blender-Porter-Pack#524]
I have a specific character that I want to append to a new scene to try making scenes with multiple characters. This one in particular (so far I only tested two) has her colors dulled down as if it applied some sort of desaturation filter. I know it must be some filter because the shader is correct in color but in the scene it isn't. If I color pick the dulled down skin color of the scene, the shader base color becomes duller, as if entering a desaturation feedback of some sort.

Original file with imported model
<img width="984" alt="Captura de pantalla 2024-02-17 a las 13 18 44" src="https://github.com/FlailingFog/KK-Blender-Porter-Pack/assets/152072467/a7df265e-26c3-4747-b2ea-2b2fb2b301a0">

Appended to new file
<img width="1029" alt="Captura de pantalla 2024-02-17 a las 13 20 29" src="https://github.com/FlailingFog/KK-Blender-Porter-Pack/assets/152072467/eed8b743-9248-469c-a25a-b6f65d649783">

**This can be avoided** by appending the new model that causes no issues to a copy of the original, but I'd like to know if this could be solvable.

**Comment by Eufalesio** [comment]
For some reason now even appending it doesn't give filter desaturation...?

**Comment by FlailingFog** [comment]
I think the default color space is set to Filmic. If you change the color space to sRGB in the new file it should match up when you append your character.

**Comment by Eufalesio** [comment]
Thanks for the tip. However, I'm using a different shader set (the one by poisenberry ) for Cycles rendering, and I don't know if this issue is still applicable to them.

**Comment by Dat-koosh** [comment]
I know this is quite a bit late and might not be relevant at all but, I noticed that the desaturation effect happens when you don't save again after trying to close the blend file of the imported model.

Basically, Blender asks you to save again regardless if you saved before or not, And this only happens the first time you import a model, Not choosing to save again will cause the model to always be desaturated whenever you open the original blend file or append them.
My guess as to why this happens is because the post shader textures are only saved when the blend file is going to be closed for whatever reason for the first time only, and you'll get the non shaded version per-se if you don't save

This pops up after you save normally and try to close the blend file and you have to click save otherwise, desaturated colors 
![image](https://github.com/FlailingFog/KK-Blender-Porter-Pack/assets/169529316/a0619665-4629-4063-8ba9-470d0322759c)

Tl;dr : Never forget to save normally and save again when you're prompted to after you import your model

Note: I have only encountered this in cycles, I have no idea if it also happens in eevee/eevee mod
